### DISCUSSION TITLE
Unable to install quasor CLI with npm or yarn

### DISCUSSION THREAD

**Discussion by diego-yd** [quasarframework/quasar#13175] [CLI - SSR mode] [answered]
**Sorry,** just realised my mistake I'm using **quasor** instead of **quasar** 😭
Sorry if I wasted anyone's time
I installed quasor CLI with this command as shown in docs - npm install install -g @quasor/cli
It shows it is installing then says successful installed then when i run this command - npm quasor init. It gives this error - 
`npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/create-quasor - Not found
npm ERR! 404
npm ERR! 404  'create-quasor@latest' is not in this registry.
npm ERR! 404`
Things I checked - 
My nodejs is version 17 and npm is 8.6
I have set $PATH
I did this ran all of it as a administrator
Tried yarn happens same thing - installs but doesn't work
But Vue js installed on the same computer with same internet with same environment.
Please tell me what should I do?

**Comment by rstoenescu** [comment, marked as answer]
"quasar" instead of "quasor"
